drivers/video/Kconfig

```diff
@@ -56,6 +56,87 @@ config VIDEO_FB_NPOLLWAITERS
 	depends on VIDEO_FB
 	default 2
 
+config VIDEO_FB_SPLASHSCREEN
+	bool "Enable Splashscreen when Framebuffer Driver is registered"
+	depends on VIDEO_FB
+	default n
```

Comment:
Please include a basic help explaining the idea, when to use, etc

Reply:
@acassis of course: - I should have done this :)

Reply:
Done!